gsoc/GSOC_2017_Application_proposal.md

```diff
@@ -0,0 +1,70 @@
+
+**Deliverables:**  
+The final result is a comprehensive input data structure and the processing module with the corresponding machine learning model. If possible, it can have a theoretical framework to deal with such kind of analogous problem.
+
+**Resources:**  
+[List any people, documentation, literature, sample data, or hardware you will need, if applicable]
+I have related knowledge of machine learning theory and algorithms. And I will pay attention to the current issue, record logs and reports, data sampled by Versionista, servers with big storage so that I can have a more intuitive undterstanding of the problem. Besides, all other potential implementations and theories. My advisor Dr. Lijun Chen and other professors, classmates are all my power to solve the problem.
+
+**Setup:**  
+I have already taken the machine learning course and the optimization course. And one project I did was the Expedia hotel recommendation which is very closed to our web monitoring project. Further more, the research I did used optimization a lot. They bring me a good sense about the machine learning algorithm and the optimization strategy. 
+
+**Ongoing involvement:**  
+Because I just joined the archivers.slack.com, I think the first thing is discussing with all members of the team and understanding their concerns or issues before. By refining the problem as clearly as possible, I will explore the monitoring data of the target website to get more intuitive understanding. Meanwhile, build the work on our current achievements. Of course, I will learn NLP and deep learning knowledge. 
+
+### Student Experience
+
+**Previous Experience:** I have a machine learning project experience: Expedia hotel recommendation. This project is the one from kaggle.com. We need to use the labeled and unlabeled data to make a prediction of the customer’s potential hotel choices. It includes to build a hotel ranking system with a prediction algorithm. I was responsible for the data processing and the construction of the ranking score. It is a very good project to implement the machine learning algorithms (e.g. KNN or linear regression) into the practical problem. 
+
+**Open Source Project(s) you are working on or would like to:** I do not have too much such kind of experiences, and this is the first open source project I would like to work on. 
+
+**Teamwork:** Most of my previous projects needed team work and collaboration. And I have played a lot of different roles in these teams. But the key point for me is following interest-oriented under the problem driven strategy. It is enjoyable to play a role in a team and figure out the answer of the problem.
+
+**Interests:** I like coding and reading. And learning by doing is the way for me to consider any problem. Besides, I like climbing and geometry.
+
+**Commitment:** I will be able to work as the required time when I am working for GSOC. The earliest emtry time is May 8th. I will fully get invovled this project.
```

Comment:
Make sure to run your proposal through a spell-checker to catch misspellings like "emtry" and "invovled".